keps/sig-auth/20190116-service-account-external-signing.md

```diff
@@ -127,9 +159,43 @@ New token generation and validation could suffer a performance difference when r
 
 Signing and verifying tokens over a grpc API carries the risk of a server side request forgery, where a malicious client could generate tokens. To mitigate this risk, the API will only be accessible over a unix socket.
```

Comment:
I think we need to expand `Risks and Mitigations` in regards to risk of API server degradation when the external signer is down (since this really is not possible with a local in-memory signer).